test/unit/helpers/xhr/v2/ENROLL_PROFILE.js

```diff
@@ -70,21 +70,6 @@ const data = {
         }
       ]
     },
-    "cancel": {
```

Comment:
Is this just a clean up?

Reply:
ah, i'll double check this. Might be able to revert.